src/lib.rs

```diff
@@ -1646,6 +1654,15 @@ impl Drop for BufferViewMut<'_> {
 }
 
 impl Buffer {
+    /// Return the bindnig view of the whole buffer.
+    pub fn as_binding(&self) -> BindingResource {
```

Comment:
This function is great but the name doesn't quite match the usage to me. It needs a `whole` or `entire` or something to show that this generates a binding for the entire buffer.